ISISMuon/MuonDataLib
# refs #10 add cython

### Description of work

This work is to move the package to use Cython and Python together. This is needed for the performance of the histogram and sorting of events. To test the Cython compilation I have added a dummy file and function (add). 

Fixes #20.

### To test:

In a clean environment use 
```
python -m pip install MuonDataLib==0.6.0b1
```

Then once its complete run the following script
```
from MuonDataLib.cython_ext.add import my_add

print(my_add(1, 2))
```
That should output 3. 

#### Code Review

- Is the code of an acceptable quality?
- Are the unit tests small and test the class in isolation?
- Does the documentation build [ReadTheDocs](https://readthedocs.org/projects/muondatalib/builds/)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**.
